rtil/src/threads/ue/rebo/rebo_init.rs

```diff
@@ -1,4 +1,5 @@
 use std::collections::HashMap;
+use std::ffi::OsStr;
```

Comment:
Now this is unused (don't ignore all warnings :) )

Reply:
Oops, just did it from my phone without thinking of warnings (hence the `Verified` tag.)